src/scripts/lsf_submit.sh

```diff
@@ -142,6 +142,31 @@ else
 fi
 # --- End of MPI directives
 
+# --- Begin of GPU directives
+
+if [ ! -z $bls_opt_gpunumber ] && [ $bls_opt_gpunumber -gt 0 ] ; then
+  if [ ! -z $bls_opt_gpumodel ] ; then
+       model="gpu_model0=='$bls_opt_gpumodel'"
+       if [ $bls_opt_gpunumber -gt 1 ] ; then
+        for i in $(seq 1 $((bls_opt_gpunumber-1)))
```

Comment:
the indentation in this block is intolerable.

Also, does the author know about this `for` loop syntax?

```suggestion
        for ((i = 1; i < bls_opt_gpunumber; i++))
```

Reply:
In Jaime's defense, both the for loop syntax and the indenting originated upstream... is it worth converting things like for-loop syntax to something different (albeit perhaps better) at the cost of making incorporating future upstream patches that much more difficult?

Reply:
I will further plead that the upstream scope also has the duplicate 'm' argument. We could drop the GPUMode attribute (and probably MICNumber as well).

Reply:
Ok - leave the indentation and for-loop like it appears in prelz's repo, if you like.